### PR TITLE
feat: update react dependencies

### PR DESCRIPTION
**Pull request by jmars**
Keep our react deps up to date so we don't have a huge upgrade later.

### CODE REVIEW

File: lib/cli/package-json.js [hunk 1]
```diff
@@ -53,7 +53,7 @@ module.exports = variables => ({
     prettier: "1.14.3",
     react: "16.6.3",
     "react-dom": "16.9.0",
```
Comment (ValGeorgiev) [inline lib/cli/package-json.js]: Can we bump react dependencies here as well. We need to bump react-dom with react as well, same with react-test-renderer

**Comment by aronshamash** [comment]
@ValGeorgiev think we should update this PR with latest dep versions as part of our perf work ?

**Comment by ValGeorgiev** [comment]
> @ValGeorgiev think we should update this PR with latest dep versions as part of our perf work ?

totally agree, it's in my to do list.

**Comment by ValGeorgiev** [comment]
I created a new PR for bumping react dependancies. I will close this one

**Pull request closed without merging**